js/views/userPageVw.js

```diff
@@ -303,10 +303,16 @@ UserPageVw = pageVw.extend({
             model.set('headerURL', self.options.userModel.get('serverUrl') + "get_image?hash=" + model.get('profile').header_hash + "&guid=" + self.pageID);
             model.set('avatarURL', self.options.userModel.get('serverUrl') + "get_image?hash=" + model.get('profile').avatar_hash + "&guid=" + self.pageID);
           }
+
           // Cache user avatar in localStorage
           var profile = model.toJSON().profile;
           window.localStorage.setItem("avatar_" + self.pageID, profile.avatar_hash);
 
+          if (response.profile.website) {
```

Comment:
short_website is from another PR that was merged, it shouldn't be showing up as a change here. It's in the PR Reed submitted.

I'll untangle the changes I made from the ones I didn't and update.